sources/packages/web/src/components/common/notes/CreateNoteModal.vue

```diff
@@ -1,44 +1,66 @@
 <template>
-  <modal-dialog-base :showDialog="showDialog" @dialogClosed="dialogClosed">
-    <template v-slot:content>
-      <v-container class="min-width-modal">
-        <formio
-          formName="createnote"
-          @loaded="formLoaded"
-          @submitted="submitNote"
-        ></formio>
-      </v-container>
-    </template>
-    <template v-slot:footer>
-      <check-permission-role :role="allowedRole">
-        <template #="{ notAllowed }">
-          <footer-buttons
-            :disablePrimaryButton="notAllowed"
-            primaryLabel="Add note"
-            @primaryClick="addNewNote"
-            @secondaryClick="dialogClosed"
-          />
-        </template>
-      </check-permission-role>
-    </template>
-  </modal-dialog-base>
+  <v-form ref="addNewNoteForm">
+    <modal-dialog-base
+      title="Create new note"
+      :showDialog="showDialog"
+      max-width="730"
+    >
+      <template #content>
+        <error-summary :errors="addNewNoteForm.errors" />
+        <div class="pb-2">
+          <span class="label-value"
+            >Add a note with relavent decisions or actions taken on this
```

Comment:
Typo on "relavent".

Reply:
Yes I saw that, but I just copied from the Figma, will check with @hellolynn-tbtb

Reply:
Thanks @andrewsignori-aot - good catch! Figma is corrected as well.